build/scripts/find-lint.sh

```diff
@@ -19,7 +19,10 @@ if [ ${1:-""} = "fast" ]
 then args="--fast"
 fi
 
-if [[ -v COMPLEMENT_LINT_CONCURRENCY ]]; then
+if [ -z ${COMPLEMENT_LINT_CONCURRENCY+x} ]; then
```

Comment:
Instead of using this empty case, I tried using `-n` but it doesn't seem to behave the same as the `-z` `else`

>    -n
>       string is not null.
>
>    -z
>      string is null, that is, has zero length
>
> https://unix.stackexchange.com/a/109631/52320


```
if [ -n ${COMPLEMENT_LINT_CONCURRENCY+x} ]; then
```

```
$ ./build/scripts/find-lint.sh
./build/scripts/find-lint.sh: line 23: COMPLEMENT_LINT_CONCURRENCY: unbound variable

$ COMPLEMENT_LINT_CONCURRENCY=1 ./build/scripts/find-lint.sh
No issues found :)
```